vcpkg/contributing/maintainer-guide.md

```diff
@@ -50,16 +52,26 @@ then obviously beneficial changes like fixing typos are appreciated!
 
 ### Check names against other repositories
 
-A good service to check many at once is [Repology](https://repology.org/).
-If the library you are adding could be confused with another one,
-consider renaming to make it clear. We prefer when names are longer and/or
-unlikely to conflict with any future use of the same name. If the port refers
-to a library on GitHub, a good practice is to prefix the name with the organization
-if there is any chance of confusion.
+Port names should, as much as possible, be unambiguous about the package the port

```

Comment:
I would split this into two paragraphs. The first paragraph should talk about our overall goal with naming and the second paragraph should talk about how well-known projects like tensorflow or zlib should take their existing well-known name.